code/modules/client/preference_setup/loadout/items/xeno/human.dm

```diff
@@ -99,3 +99,57 @@
 	sort_category = "Xenowear - Human"
 	flags = GEAR_HAS_DESC_SELECTION
 	origin_restriction = list(/singleton/origin_item/origin/coa_spacer)
+
+/datum/gear/head/human
+	display_name = "human hat selection"
+	sort_category = "Xenowear - Human"
+
+/datum/gear/head/human/New()
+	..()
+	var/list/hats = list()
+	hats["hat, conical"] = /obj/item/clothing/head/nonla
+	hats["hat, gat"] = /obj/item/clothing/head/konyang
+	hats["hat, himean cap"] = /obj/item/clothing/head/softcap/himeo
+	hats["hat, hachimaki"] = /obj/item/clothing/head/hachimaki
+	hats["hat, fisanduhian ushanka"] = /obj/item/clothing/head/ushanka/dominia
+
+/datum/gear/head/vysoka
+	display_name = "vysokan hat selection"
+	description = "A selection of vysokan hats."
+	path = /obj/item/clothing/head/vysoka
+	flags = GEAR_HAS_DESC_SELECTION
+	origin_restriction = list(/singleton/origin_item/origin/vysoka, /singleton/origin_item/origin/ipc_vysoka)
+	sort_category = "Xenowear - Human"
+
```

Comment:
this is a problem with what you're doing - multiple species can use multiple human loadout items. e.g unathi can use dominian loadout items, IPCs can use konyang head items, tajara can use himean items and that's just a few off the top of my list.

this is why I'm not particularly sure on the xenowear - human change, it's really inconsistent

Reply:
my main modus operandi for organizing is that if it's mainly in the purview of human lore and/or majority humans using the item, it's going into Xenowear - Human. ideally the "generic" main loadout section should be able to be used by all or at least a majority of characters, while any human-only or human-plus-one goes to Xenowear-Human

and also makes it easier to tell which species lore sections are in charge of adding what items into the game and who's responsible for it